src/NuGet.Core/NuGet.Protocol/Model/PackageDeprecationReason.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+namespace NuGet.Protocol.Model
```

Comment:
Was this intended to be in Protocol? If so, can you share more about what changes you'll be making to Protocol?

Reply:
We have a similar enum for vulnerabilities, PackageVulnerabilitySeverity, I believe this is a good place to have this enum since it's really similar but at the cost of creating new public APIs. 

I don't plan any changes to the protocol in this effort, do you believe this shouldn't be here? Should I move this to the UI projects?

Reply:
My general thoughts are putting model code into the UI projects makes sense. If it's needed later in Protocol, it will need to be brought over, anyway.

Reply:
I think it's unusual to have a type defined in a class but never used in it. 

I agree with @donnie-msft, we should move this to the UI project.